frontends/bmd/src/pages/admin_screen.tsx

```diff
@@ -71,6 +75,100 @@ export function AdminScreen({
     fetchElection();
   }
 
+  useEffect(() => {
```

Comment:
this brute-forcing of strings to pre-warm the cache is... not my favorite approach. But, it has the virtue of being simple, not super likely to fail, and if it fails, the worst outcome is some strings aren't cached and will be when they're later spoken.